docs/pages/access-controls/guides/device-trust.mdx

```diff
@@ -137,17 +152,55 @@ Relogin to fetch updated certificates:
 $ tsh logout; tsh login --proxy=example.com --user=alice
 ```
 
-Having the necessary permissions, let's add a device to the inventory. Replace
-the `$SERIAL` below with your macOS device serial number (visible under Apple ->
-"About This Mac" -> "Serial number", or use the command below).
+Before we can enroll the device, we need to register it. In order to register
+a device, we need to first determine its serial number.
+
+<Tabs>
+  <TabItem label="macOS">
+    The serial number of a macOS device can be determined in two ways.
+
+    Through the UI, visible under Apple -> "About This Mac" -> "Serial number".
+
+    Through the terminal:
+    ```code
+    $ ioreg -c IOPlatformExpertDevice -d 2 | grep -i IOPlatformSerialNumber | awk -F'"' '{print $4}'
+    (=devicetrust.asset_tag=)
+    ```
+  </TabItem>
+
+  <TabItem label="Windows">
+    Determining the serial number for a device running Windows is a little more
+    complicated than on macOS. This is because Windows devices can have multiple
+    serial numbers depending on the configuration made by the manufacturer.
+
+    Teleport will pick the first available value from the following:
+
+    - System asset tag
+    - System serial number
+    - Baseboard serial number
+
+    To find the value chosen by Teleport, run the following command:
```

Comment:
It's a little awkward because of the fallback behaviour. We could list all three powershell commands, I'm a little less sure about the UI location for some of these. I wasn't able to find the system asset tag 😓

Reply:
`tsh device serial-number` makes this moot, so no worries.